### ISSUE TITLE
[Feature Request] Import GraphQL schema as collection

### ISSUE THREAD

**Issue by Techbrunch** [getinsomnia/insomnia#2045]
It would be awesome if there was the possibility to generate a collection of requests from a GraphQL schema if introspection is enabled.

Something like this: https://github.com/oshosanya/graphql-export/blob/master/src/exporters/insomnia/insomnia.ts

**Comment by develohpanda** [comment]
Wow, that is awesome! It may be possible to do something similar via an Insomnia plugin in the near future! 😁

**Comment by mehdibahrami** [comment]
Postman support this one but in a so bad model and because of that I switched to Insomnia and I saw you don't support it!
It's really simple just user select a file and u make the requests based on the GraphQL Schema and also he should have an ability to refresh the collection and the app add new requests based on that

**Comment by JamesPeters98** [comment]
Is this feature a possibility at all? It's the only thing stop me from switching from Postman. I'd ideally be able to generate queries and mutations with all the variables just like in Postman. And also be able to version each collection, as the API evolves.

**Comment by myspivey** [comment]
Comment to keep alive, still very interested in this.

**Comment by staff0rd** [comment]
This would be epic

**Comment by osecluna** [comment]
Yes please! It would be good it it could work something like importing a rest API from the openAPI spec.

**Comment by shlomich** [comment]
Is this out?

**Comment by M-Davies** [comment]
If this is still alive, bump please. Especially after Postman's recent enforcement of cloud sync'd collections
